### PR TITLE
[EventGhost] - Upgrade - Stackless Python to 2.7.13150

### PR DESCRIPTION
**Pull request by kdschlosser**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -193,19 +193,19 @@ notifications:
 
 
     Artifacts:
-    
+
```
Comment (kdschlosser) [inline .appveyor.yml]: Cleared white space IDE did it

File: .appveyor.yml [hunk 2]
```diff
@@ -193,19 +193,19 @@ notifications:
...
     {{#jobs}}
     {{#artifacts}}
     [{{name}}]({{url}})
-    
+
```
Comment (kdschlosser) [inline .appveyor.yml]: Cleared white space IDE did it

File: .appveyor.yml [hunk 3]
```diff
@@ -193,19 +193,19 @@ notifications:
...
     {{/artifacts}}
     {{/jobs}}
 
   on_build_success: true
   on_build_failure: true
   on_build_status_changed: false
-  
-  
+
+
```
Comment (kdschlosser) [inline .appveyor.yml]: Cleared white space IDE did it

**Comment by kdschlosser** [comment]
You will have to clear the appveyor cache for this to work properly. You will also need to clear it again after it is done building.

File: .appveyor.yml (hunk 1 above)
Comment (topic2k) [inline .appveyor.yml]: Did what? Has it to do with this PR or does it fix something else?

**Comment by topic2k** [comment]
The cache should automatically be cleared when .appveyor.yml changes.

**Comment by kdschlosser** [comment]
i now believe we have a problem with code climate.. i broke it.. LOL

**Comment by topic2k** [comment]
Aaargh, is EG not enough for you to break? What next? Appveyor? Python? The World? 😂 🤣  😆

**Comment by kdschlosser** [comment]
I know right.

File: .appveyor.yml (hunk 1 above)
Comment (kdschlosser) [inline .appveyor.yml]: no there was white space on multiple lines in this file. so when i opened it in my IDE and made the changes then saved it. the IDE will automatically remove any unneeded white space.

**Comment by kdschlosser** [comment]
Not going to use. Will wait until the fix for Cython is included in a release.

**Pull request closed without merging**